pkg/vpa/vpa.go

```diff
@@ -348,6 +367,7 @@ func (r Reconciler) getVPAObject(existingVPA *vpav1.VerticalPodAutoscaler, ns *c
 				Name:      "goldilocks-" + controller.Name,
 				Namespace: ns.Name,
 			},
+			Spec: vpav1.VerticalPodAutoscalerSpec{},
```

Comment:
is this added for a particular reason?

Reply:
Let me double check. I recall adding it when I added the label functionality, but it may have been a faulty test so Ill address in my second rev of this